### PR TITLE
docs(example): Add Chackra UI

### PR DESCRIPTION
**Pull request by archwebio**
Creating a basic example with Chackra UI

### CODE REVIEW

**Comment by juwiragiye** [comment]
@archwebio can you add a  CodeSandbox link. I want to try this example because I've been having problems with Chackra UI

File: examples/chakra-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/chakra-ui)
+
+## Example
+
+This example shows how to use Chakra UI with Remix.
+
+Check [app/root.tsx](./app/root.tsx) where Chakra UI is imported and providing context to the component tree.
+
+> You'll need to setup the `ChakraProvider` in every other place when the full `Document` gets replaced as in `ErrorBoundary` and `CatchBoundary`.
```
Comment (kentcdodds) [inline examples/chakra-ui/README.md:19]: I think this comment is misleading without an example. Could you demonstrate what you mean by including a boundary in the example code? Then the content can be removed from here.

```suggestion
```

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Great start! One minor change needed.

**Comment by archwebio** [comment]
> @archwebio can you add a CodeSandbox link. I want to try this example because I've been having problems with Chackra UI

Yes, I'll setup an example for you to test.

Can you elaborate on the issues you have experienced? It was with some specific Chackra UI API?

File: examples/chakra-ui/README.md (hunk 1 above)
Comment (archwebio) [reply examples/chakra-ui/README.md]: Great point

**Review by kentcdodds — APPROVED** [review]
Stellar! Thanks 👏

**Pull request merged**